src/cpp/src/visual_language/inputs_embedder.hpp

```diff
@@ -128,6 +134,9 @@ class InputsEmbedder {
         // True, if m_add_special_tokens was set, otherwise default behaviour is used
         bool m_add_special_tokens_is_set = false;
         virtual ~IInputsEmbedder() = default;
+        // position ids
+        ov::Tensor m_position_ids;
+        int64_t m_rope_delta = 0;
```

Comment:
Please keep class members before destructor
```suggestion
        // position ids
        ov::Tensor m_position_ids;
        int64_t m_rope_delta = 0;
        
        virtual ~IInputsEmbedder() = default;
```